Fixing PR builder errors for PR 3141 (#3153)
SumoLogic/sumologic-documentation: docs/integrations/microsoft-azure/azure-api-management.md

---
id: azure-api-management
title: Azure API Management
description: Learn about the Sumo Logic collection process for the Azure API Management service.
---

import useBaseUrl from '@docusaurus/useBaseUrl';

<img src={useBaseUrl('img/integrations/microsoft-azure/azure-api-management.png')} alt="Thumbnail icon" width="50"/>

> [Azure API Management](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts) is a hybrid, multicloud management platform for APIs across all environments. As a platform-as-a-service, API Management supports the complete API lifecycle. This integration provides near-real-time visibility into the state and health of your APIs, along with rich information about API Management operations that are important for auditing.

Reviewer: ```suggestion
[Azure API Management](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts) is a hybrid multicloud management platform for APIs across all environments. As a platform-as-a-service, API Management supports the complete API lifecycle. This integration provides near-real-time visibility into the state and health of your APIs, along with rich information about API Management operations that are important for auditing.
```

> [Azure API Management](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts) is a hybrid, multicloud management platform for APIs across all environments. As a platform-as-a-service, API Management supports the complete API lifecycle. This integration helps in giving you near real-time visibility into the state and health of your APIs and also provides rich information about API Management operations that are important for auditing.

Reviewer 2: ```suggestion
[Azure API Management](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts) is a hybrid, multicloud management platform for APIs across all environments. As a platform-as-a-service, API Management supports the complete API lifecycle. This integration provides near-real-time visibility into the state and health of your APIs, along with rich information about API Management operations that are important for auditing.
```
